include/vctr/Expressions/ExpressionChainBuilder.h

```diff
@@ -25,13 +25,68 @@ namespace vctr
 
 namespace detail
 {
+/** Internal helper structure to pass runtime arguments to expressions in a chain
+    built from a concatenation of expression chain builder instances.
+
+    The expression chain builder primarily holds a template template for the
```

Comment:
a template template? 😄

Reply:
Yes, indeed "template template" is an term used by the community, see e.g. the section on it on [cpprefence](https://en.cppreference.com/w/cpp/language/template_parameters)
![Screenshot 2023-01-10 at 11 49 59](https://user-images.githubusercontent.com/13800135/211532031-3c6c4064-0e13-401c-9a80-b3c405e1a3fc.png)